chains/berachainbartio/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.routescan.io/v2/network/testnet/evm/80084/etherscan/api/
+    family: routescan
+    name: Bartio Testnet Explorer
+    url: https://bartio.beratrail.io/
+chainId: 80084
+deployer:
+  name: Henez Foundation
+  url: https://www.henez.fi/
+displayName: Berachain bArtio
+domainId: 80084
+isTestnet: true
+name: berachainbartio
+nativeToken:
+  decimals: 18
+  name: BERA
+  symbol: BERA
+protocol: ethereum
+rpcUrls:
+  - http: https://bartio.rpc.berachain.com/
```

Comment:
Is there a second rpc url we can add here as a fallback?

Reply:
Yes, I have added more rpc url